Makefile

```diff
@@ -41,8 +41,8 @@ tag: ## Creates release tag
 .PHONY: clean
 clean: ## Cleans go and generated files in ./dapr/proto/
 	go clean
-	rm -fr ./dapr/proto/common/v1/*
-	rm -fr ./dapr/proto/runtime/v1/*
+	rm -fr ./dapr/proto/common/v1/*.pb.go
```

Comment:
This is needed because I've added a non-generated file in those folders, with code that has to be crafted by hand.

Reply:
no new file under common, do I miss it?